c172p-fdm-dany93.xml

```diff
@@ -939,13 +950,23 @@
                     <value>0.0147</value>
                 </product>
             </function>
-            <function name="aero/coefficient/Clbk">
-                <description>Roll_moment_due_to_one_broken wing</description>
+            <function name="aero/coefficient/Cllwdmg">
```

Comment:
Are `aero/coefficient/Cllwdmg` and `aero/coefficient/Clrwdmg` summed and used somewhere?